### PR TITLE
Feature/parent data modelling

### PR DESCRIPTION
**Pull request by bramirez96**
# Description

Endpoints, database queries, and tests for Parent data.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by bramirez96** [comment]
[WEB Task - As a user, I can log in and sign up](https://trello.com/c/siDek1eA/4-web-task-as-a-user-i-can-log-in-and-sign-up)

**Review by Nick-Ohman — DISMISSED** [review]
The work looks great!
Nice documentation.

**Review by JLC6290 — COMMENTED** [review]
Great commit messages, documentation is solid.

File: api/parent/parentRouter.js [hunk 1]
```diff
@@ -0,0 +1,357 @@
+const express = require('express');
+const authRequired = require('../middleware/authRequired');
+const Parents = require('./parentModel');
+const router = express.Router();
+
+/**
+ * @swagger
```
Comment (Schrese) [inline api/parent/parentRouter.js:7]: These swagger docs look great!

**Review by Schrese — DISMISSED** [review]
Might be good to put some of the instructions for future people who want to clone and start up the databases locally. Not an immediate thing, just something to think about. Overall this looks great!!!

**Review by SandraCoburn — DISMISSED** [review]
Great documentation and tests. Explicit comments when needed. Clean code.

**Review by Schrese — APPROVED** [review]
Oh, excellent! Thank you for adding the information in the readme

**Review by JLC6290 — APPROVED** [review]
Everything looks good, swaggerdocs look great!

**Review by SandraCoburn — APPROVED** [review]

**Pull request merged**